sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -243,8 +251,8 @@ where
     // Build an Invocation holding the two channels and send it down the
     // specified channel.
     let invocation = Invocation {
-        request_receiver: req_receiver.clone(),
-        response_sender: rsp_sender.clone(),
+        receiver: Some(req_receiver.clone()),
```

Comment:
Does `GrpcInvocationSender` check that both fields are not `None`? Since it will be an internal error

Reply:
There is not a `send` function on GrpcInvocationSender. Its WriteHandle is extracted and used in `get_invocation_channel` and an Error is returned there if the `sender` is `None`.